cmd/podman/registry/config.go

```diff
@@ -16,6 +16,7 @@ import (
 
 const (
 	ParentNSRequired = "ParentNSRequired"
+	UserNSRequired   = "UserNSRequired"
```

Comment:
Could this be renamed? Maybe "RootRequired"?

Reply:
I don't think that is helpful.  How about UnshareUserNSRequired Or UnshareNSRequired

Reply:
UnshareNSRequired SGTM